src/CodeSigning/CodeSigning/CodeSigning.csproj

```diff
@@ -24,32 +24,33 @@
 
   <ItemGroup>    
     <PackageReference Include="Moq" Version="4.18.4" />
-    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="7.0.1" />    
+    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="6.0.2" />
```

Comment:
Can't depend on System.Security.Cryptography.Pkcs v7 because it requires a higher dependency of System.Memory than Azure PowerShell has. v6 is OK